### PR TITLE
Add load_test.py as a CLI tool

### LINKED ISSUE #570: Support some basic load-testing
Maybe with e.g. locust? @milesgranger  played a bit with it before, but it might need a bit of update to support new api (?). I dont know if it belongs in this repo, and I dont think it needs to be a big thing, but it would be useful to be able to play with different settings on the server and see how requests per second we can get it to.

### PR DESCRIPTION
**Pull request by EinarJohnsen**
Add a load test CLI tool to use Locust to simulate users and load on the servers. This can be used to test how the servers respond to many requests as well as how the cluster handles autoscaling. Fixes #570

### CODE REVIEW

**Comment by EinarJohnsen** [comment]
I think this is good enough to start using if you want to load test.

File: benchmarks/load_test/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#### Usage
+
+- Start port-forwarding with *ambassador* on a cluster (with an existing gordo-project).
+
+    `kubectl port-forward svc/ambassador -n ambassador 8888:80`
+
+- Use the CLI tool `load_test.py`, specify the *project name* (required), *host* (optional) and *port* (optional).
+***Note** the port needs to match the port which is port-forwarded to*
+
+- Go to: [locust instance](http://127.0.0.1:8089) and specify the number of users to simulate, as well as the the number of users
```
Comment (milesgranger) [inline benchmarks/load_test/README.md]: I like the link, but should perhaps be [locust instance: http://127.0.0.1:8089](http://127.0.0.1:8089) just so it's explicit _where_ the default server is.

File: benchmarks/load_test/load_test.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import yaml
+from collections import namedtuple
+
+import numpy as np
+import jinja2
+import click
+from locust import HttpLocust
+
+TEMPLATE_FILE = "task_set.py.jinja2"
+Action = namedtuple("Action", "name path json")
```
Comment (milesgranger) [inline benchmarks/load_test/load_test.py]: I guess I probably wrote this originally, but do you think it should be named `Task` instead, isn't that what locust refers to these things as? :man_shrugging:

File: benchmarks/load_test/load_test.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            "tag_list", []
+        )
+        target_tag_list = element["endpoint-metadata"]["metadata"]["dataset"].get(
+            "target_tag_list", []
+        )
+
+        tags[name] = {
+            "tag_list": len(tag_list),
+            "target_tag_list": len(target_tag_list),
+        }
```
Comment (milesgranger) [inline benchmarks/load_test/load_test.py:35]: If you have access to `gordo_components` you could use something like this:
```python
from gordo_components.client.utils import EndpointMetadata

meta = EndpointMetadata(element)
meta.name
meta.target_tag_list
...
```
And if the metadata changes, no one has to remember that this code would also break. Or at least less likely to break?

File: benchmarks/load_test/task_set.py.jinja2 [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+class Tasks(TaskSet):
+    """
+    Generated locust TaskSet where methods/tasks are generated for each
+    gordo model listed in the project's config file.
+    """
+{% for action in actions %}
+    @task
+    def {{ action.name }}(self):
+        self.client.post("{{ action.path }}", json={{ action.json }})
+{% endfor %}
```
Comment (milesgranger) [inline benchmarks/load_test/task_set.py.jinja2]: New line

File: benchmarks/load_test/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+It *requires* locust to be installed, use `pip install locustio` (if dev-dependencies is not already installed).
+
+#### Usage
+
+- Start port-forwarding with *ambassador* on a cluster (with an existing gordo-project).
+
+    `kubectl port-forward svc/ambassador -n ambassador 8888:80`
+
+- Use the CLI tool `load_test.py`, specify the *project name* (required), *host* (optional) and *port* (optional).
```
Comment (milesgranger) [inline benchmarks/load_test/README.md:13]: Should you add a short little example snippet here? something lazy people like myself can copy and paste :-)

**Review by milesgranger — COMMENTED** [review]
Looking pretty good from my end, handful of comments. See if you agree with them. :stuck_out_tongue_winking_eye:

**Review by milesgranger — APPROVED** [review]

**Pull request merged**